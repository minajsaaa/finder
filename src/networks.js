const networks = [
  {
    value: "columbus-1",
    key: "Columbus-1",
    lcd: "https://lcd.terra.dev",
    rpc: "https://rpc.terra.dev",
    wss: "ws://rpc.terra.dev/websocket"
  },
  {
    value: "soju-0008",
    key: "Soju-0008 Testnet",
    lcd: "https://soju-0008-lcd.terra.dev",
    rpc: "https://soju-0008-rpc.terra.dev",
    wss: "ws://soju-0008-rpc.terra.dev/websocket"
  }
];

export const getNetwork = network =>
  networks.find(({ value }) => value === network) || {};

export default networks;
