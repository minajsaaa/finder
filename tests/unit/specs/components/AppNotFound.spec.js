import networks from "@/networks";
import { mount } from "@vue/test-utils";
import AppNotFound from "@/components/AppNotFound";

describe(`AppNotFound`, () => {
  let $route, wrapper;

  beforeEach(() => {
    $route = {
      params: {
        network: networks[0].value,
        keyword: "xxxx"
      }
    };

    wrapper = mount(AppNotFound, {
      mocks: {
        $route
      },
      stubs: [`router-link`]
    });
  });

  it(`AppNotFound Snapshot`, () => {
    expect(wrapper.vm.$el).toMatchSnapshot();
  });
});
