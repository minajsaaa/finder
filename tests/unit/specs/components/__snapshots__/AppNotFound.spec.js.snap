// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppNotFound AppNotFound Snapshot 1`] = `
<div
  class="not-found"
>
  <div>
    <h1>
      Search not found
    </h1>
    <p>
      Sorry, we couldn't find any results for 
      <span>
        "xxxx"
      </span>
      .
    </p>
    <p>
      Please input the correct block number or transaction hash or account address.
    </p>
    <img
      src="https://s3.ap-northeast-2.amazonaws.com/terra.money.home/static/finder/earth.svg"
    />
    <router-link-stub
      exact="exact"
      to="/"
    >
      <span>
        Back to Home
      </span>
    </router-link-stub>
  </div>
</div>
`;
